src/server/server_state.cc

```diff
@@ -71,6 +71,15 @@ void ServerState::Destroy() {
   state_ = nullptr;
 }
 
+uint64_t ServerState::GetCachedUsedMemory(uint64_t now_ns) {
```

Comment:
Why not calculate `now_ns` inside the method? Is it due to performance reasons?
It looks like now we call it once per `DispatchCommand()`, so it shouldn't be noticeable (right?)

Reply:
I actually think it's better to pass time variables for several reasons. 
1. It's usually much harder to test code the internally relies on clock calls.
2. Once the default is flipped and we declare it's better style calling the clock internally - the code is immediately flooded with clock calls.
3. Which brings us to the biggest thing - clock calls can be expensive. It depends on the hardware, virtualisation etc but ask @dranikpg  - i remember that at some point all the benchmarks he did on his laptop were meaningless because the clock function took like 30% of the total CPU in Dragonfly.

Reply:
Sure, then feel free to ignore